### PR TITLE
Feature/10760 2 g plus check in/10875 certificate switch

### PR DESCRIPTION
**Pull request by nickguendling**
## Description
Implements the new design changes for the certificates tab for different admission states, including the option to switch between different certificates for 2G+.

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10875

## Screenshots

IMG_0223
IMG_0224
IMG_0225
IMG_0226
IMG_0227
IMG_0228
IMG_0229
IMG_0230
IMG_0231
IMG_0232
IMG_0233
IMG_0234
IMG_0235
IMG_0236
IMG_0243
IMG_0244
IMG_0245
IMG_0246
IMG_0247
IMG_0248

### CODE REVIEW

**Review by CV113 — APPROVED** [review]
UA approved

File: src/xcode/ENA/ENA/Source/Views/GradientView.swift [hunk 1]
```diff
@@ -10,25 +10,30 @@ class GradientView: UIView {
 	// MARK: - Init
 
 	init(
-		type: GradientType = .solidGrey(withStars: false),
+		type: GradientType = .solidGrey,
+		withStars starsAreVisible: Bool = false,
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Views/GradientView.swift:14]: 👍

**Review by naveeddotio — APPROVED** [review]
Not tested on iOS 12 device.

File: src/xcode/ENA.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 2]
```diff
@@ -154,6 +154,15 @@
           "version": "4.6.0"
         }
       },
+      {
+        "package": "swift-collections",
```
Comment (marcussc) [inline src/xcode/ENA.xcworkspace/xcshareddata/swiftpm/Package.resolved:158]: This requires an update of the licenses.json in the root of the project followed by `bundle exec fastlane update_licenses`

**Review by marcussc — CHANGES_REQUESTED** [review]
Runs on my 12.5.3 device :)

**Review by 30mar — APPROVED** [review]

**Review by marcussc — APPROVED** [review]

**Pull request merged**